frontend/mr-admin-flate/src/components/detaljside/Metadata.tsx

```diff
@@ -21,8 +22,10 @@ export function Metadata({
 }) {
   return (
     <div className={styles.header_og_verdi}>
-      <dt className={styles.bold}>{header}</dt>
-      <dd className={styles.definition}>{verdi}</dd>
+      <Heading level="3" size="xsmall">
+        {header}
+      </Heading>
```

Comment:
Uavhengig av hva vi lander på ref. kommentaren over så tror jeg ikke nødvendigvis heading 3 er korrekt semantisk. Det blir veldig mange headers på siden for en eventuell skjermleser.